src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -150,6 +150,35 @@
  * <pre>
  * &lt;module name="JavadocMethod"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * &#47;**
+ *  * A sample function to add two numbers
+ *  * @param number1 The first number
+ *  * @param number2 The second number
+ *  * @return An integer
+ *  *&#47;
```

Comment:
Please remove the summary i.e. `A sample function to add two numbers` and just keep the tags as CI is failing due to it. (For All)

```
...
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java:239: The Javadoc block tag '@param' should be placed at the beginning of the line. [JavadocBlockTagLocation]
...
```